src/components/tree/phyloTree/helpers.js

```diff
@@ -77,26 +83,61 @@ function _getSpaceBetweenSubtrees(numSubtrees, numTips) {
  * PhyloTree can subsequently use this information. Accessed by prototypes
  * rectangularLayout, radialLayout, createChildrenAndParents
  * side effects: <phyloNode>.displayOrder (i.e. in the redux node) and <phyloNode>.displayOrderRange
- * @param {Array<PhyloNode>} nodes
+ * @param {Object} props
+ * @param {Array<PhyloNode>} props.nodes
+ * @param {boolean} props.focus
  * @returns {undefined}
  */
-export const setDisplayOrder = (nodes) => {
+export const setDisplayOrder = ({nodes, focus}) => {
+  timerStart("setDisplayOrder");
+
   const numSubtrees = nodes[0].n.children.filter((n) => n.fullTipCount!==0).length;
-  const numTips = nodes[0].n.fullTipCount;
+  const numTips = focus ? nodes[0].n.tipCount : nodes[0].n.fullTipCount;
   const spaceBetweenSubtrees = _getSpaceBetweenSubtrees(numSubtrees, numTips);
+
+  // No focus: 1 unit per node
+  let incrementer = (_node) => 1;
+
+  if (focus) {
+    const numVisible = nodes[0].n.tipCount;
+
+    let yProportionFocused = 0.8;
+    // Adjust for a small number of visible tips (n<4)
+    yProportionFocused = Math.min(yProportionFocused, numVisible / 5);
+    // Adjust for a large number of visible tips (>80% of all tips)
+    yProportionFocused = Math.max(yProportionFocused, numVisible / nodes.length);
```

Comment:
`nodes` includes internal nodes whereas we just want the proportion of tips, so this should be `numVisible / nodes[0].n.fullTipCount`

Reply:
Good catch! Fixed and used a new variable for clarity: https://github.com/nextstrain/auspice/blob/126b895d6cc0623b6a1b5534873291f6d8977bd3/src/components/tree/phyloTree/helpers.js#L102-L112